WalletLibrary/Submodules/VerifiableCredential-SDK-iOS/VCNetworking/VCNetworking/encoders/PresentationResponseEncoder.swift

```diff
@@ -31,13 +23,19 @@ struct PresentationResponseEncoder: Encoding {
         
         let idTokenParam = "\(Constants.idToken)=\(try value.idToken.serialize())"
         
-        guard let vpToken = value.vpToken else {
+        var vpTokenParam = ""
+        
+        guard !value.vpTokens.isEmpty else
+        {
             throw PresentationResponseEncoderError.noVerifiablePresentationInResponse
         }
         
-        let vpTokenParam = "\(Constants.vpToken)=\(try vpToken.serialize())"
+        for vpToken in value.vpTokens
+        {
+            vpTokenParam = vpTokenParam + "&\(Constants.vpToken)=\(try vpToken.serialize())"
```

Comment:
This might be an iOS thing. Does the response body have to be created manually?

Reply:
yeah :/